### PR TITLE
Bump Up Versions Post Release

### PR DESCRIPTION
**Pull request by chidozieononiwu**

### CODE REVIEW

File: sdk/storage/Azure.Storage.Files.DataLake/src/Azure.Storage.Files.DataLake.csproj [hunk 1]
```diff
@@ -4,7 +4,7 @@
   </PropertyGroup>
   <PropertyGroup>
     <AssemblyTitle>Microsoft Azure.Storage.Files.DataLake client library</AssemblyTitle>
-    <Version>12.0.0-preview.5</Version>
+    <Version>12.0.0-preview.6</Version>
```
Comment (tg-msft) [inline sdk/storage/Azure.Storage.Files.DataLake/src/Azure.Storage.Files.DataLake.csproj]: Please revert this one - it hasn't been released yet and we'll want `.5` when they do.

**Review by tg-msft — APPROVED** [review]

File: sdk/storage/Azure.Storage.Files.DataLake/src/Azure.Storage.Files.DataLake.csproj (hunk 1 above)
Comment (chidozieononiwu) [reply sdk/storage/Azure.Storage.Files.DataLake/src/Azure.Storage.Files.DataLake.csproj]: Oops. Right. it hasn't been released. Reverting. Thanks.

**Review by heaths — APPROVED** [review]
LGTM for Key Vault

**Review by annelo-msft — APPROVED** [review]
+1 for AppConfig.

File: sdk/core/Microsoft.Extensions.Azure/src/Microsoft.Extensions.Azure.csproj [hunk 2]
```diff
@@ -4,7 +4,7 @@
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
     <Description>Azure Client SDK integration with Microsoft.Extensions libraries</Description>
     <AssemblyTitle>Azure Client SDK integration Microsoft.Extensions</AssemblyTitle>
-    <VersionPrefix>1.0.0</VersionPrefix>
+    <VersionPrefix>1.1.0-preview.1</VersionPrefix>
```
Comment (weshaggard) [inline sdk/core/Microsoft.Extensions.Azure/src/Microsoft.Extensions.Azure.csproj]: We probably need to do a pass over the repo and change any of these `VersionPrefix` properties to `Version`

Comment (chidozieononiwu) [reply sdk/core/Microsoft.Extensions.Azure/src/Microsoft.Extensions.Azure.csproj]: I'll do that

**Pull request merged**